src/components/data/PathListAutomation.js

```diff
@@ -193,246 +199,292 @@ export default function PathListAutomation(props) {
-                                        components={{
-                                            pattern: (
-                                                <ExternalLink
-                                                    href={
-                                                        constants.JAVA_PATTERN_DOC
-                                                    }
-                                                />
-                                            ),
-                                        }}
-                                    />
-                                </Typography>
 
-                                <Field
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_MATCH_PATTERN
-                                    )}
-                                    name="pattern"
-                                    component={FormTextField}
-                                    placeholder="e.g: \.csv$"
-                                    variant="outlined"
-                                    dense
-                                />
-                            </Grid>
-                            <Grid item xs>
-                                <Typography variant="body2">
-                                    {t("pathListInfoTypeLbl")}
-                                </Typography>
-                                <Paper>
-                                    <List
-                                        style={{
-                                            maxHeight: 150,
-                                            overflow: "auto",
-                                        }}
-                                        id={buildId(
-                                            baseId,
-                                            ids.PATH_LIST_AUTO_MATCH_INFO_TYPES
-                                        )}
-                                    >
-                                        {isFetching && (
-                                            <Skeleton
-                                                variant="rect"
-                                                height={150}
-                                            />
-                                        )}
-                                        {!isFetching &&
-                                            infoTypes &&
-                                            infoTypes.length > 0 &&
-                                            infoTypes.map((type) => {
-                                                const labelId = `checkbox-list-label-${type}`;
-                                                return (
-                                                    <ListItem
-                                                        key={type}
+                                    <DialogContent>
+                                        <>
```

Comment:
This Fragment isn't necessary (`DialogContent` can already contain multiple children).